server/src/core/server/services/comments/actions.ts

```diff
@@ -353,6 +355,20 @@ export async function createReaction(
     ).catch((err) => {
       logger.error({ err }, "could not publish comment flag created");
     });
+
+    if (externalNotifications.active()) {
+      const parentAuthor = comment.authorID
```

Comment:
Using `parentAuthor` and `author` here for recs doesn't seem as clear to me as it does for reply notifications. What about something more like `commentAuthor` and `recAuthor` or `recUser` or `userWhoRecced`?